.golangci.yaml

```diff
@@ -11,6 +11,8 @@ linters:
 
 run:
   timeout: 10m
+  build-tags:
+    - codeanalysis
```

Comment:
nit: The indentation doesn't match the rest of the file.

About the tag: `codeanalysis` feels obscure to me. How about something that says exactly what this is about, such as `noclibs` ("no C libraries"). It's more versatile and can be used with anything (not just analysis tools).